vector/build.gradle

```diff
@@ -85,7 +85,6 @@ android {
                 testCoverageEnabled = coverage.enableTestCoverage
             }
         }
-<<<<<<< HEAD
         // Tchap: No nightly
 //        nightly {
```

Comment:
We can probably remove the nightly bloc if this is not present anymore in this gradle file in Element